src/transformStateWithClones.js

```diff
@@ -8,6 +8,32 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+
+  let copy = { ...state };
+  const story = [];
+
+  actions.forEach((action) => {
+    switch (action.type) {
```

Comment:
[CODE STYLE]: Use a `switch` statement when you have a limited amount of conditions, which you have done correctly here.